shardingsphere-distsql-parser/shardingsphere-distsql-parser-engine/src/main/antlr4/imports/RDLStatement.g4

```diff
@@ -58,3 +58,11 @@ user
 password
     : IDENTIFIER | INT | STRING
     ;
+
+connectionProperties
+    : connectionProperty (COMMA connectionProperty)*
+    ;
+
+connectionProperty
+    : key=IDENTIFIER EQ value=IDENTIFIER
```

Comment:
Other `g4` file  define  properties as follows:
```
key=(IDENTIFIER | STRING) EQ value=(NUMBER | INT | STRING)
```

please confirm whether need to consider consistency.

Reply:
`key=(IDENTIFIER | STRING) EQ value=(NUMBER | INT | STRING)`
Can not match the following statement: 
```
"useSSL"="false"
```

Reply:
We may update to `key=(IDENTIFIER | STRING) EQ value=(IDENTIFIER | STRING)`